src/pages/seonghwanCho/Main/Main.js

```diff
@@ -1,7 +1,110 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Nav from '../../../components/Nav/Nav';
+import SidebarRecomUser from '../../../components/seonghwanCho/SidebarRecomUser';
+import ArticleStory from '../../../components/seonghwanCho/ArticleStory';
+import Feed from '../../../components/seonghwanCho/Feed';
+import './Main.scss';
 
-function Main() {
-  return <div>Nav bar</div>;
+class Main extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      feedList: [],
+      commentValue: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/commentData_seonghwan.json')
```

Comment:
```suggestion
    fetch('/data/commentData_seonghwan.json')
```
localhost:3000은 생략가능합니다

Reply:
감사합니다! 수정 했습니다!